File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { AareguruSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = AareguruSDK.test()
    equal(testsdk instanceof AareguruSDK, true,
      'AareguruSDK.test() must return a client synchronously')
  })

})

File: rb/test/exists_test.rb
# Aareguru SDK exists test

require "minitest/autorun"
require_relative "../Aareguru_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = AareguruSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
